test/e2e-common.sh

```diff
@@ -46,6 +46,11 @@ function install_pipeline_crd() {
   wait_until_pods_running tekton-pipelines || fail_test "Tekton Pipeline did not come up"
 }
 
+function delete_pipeline_crd() {
+  echo ">> Deleting Tekton Pipelines"
+  kubectl delete --filename https://github.com/tektoncd/pipeline/releases/download/v0.11.0/release.yaml || fail_test "Tekton pipeline deletion failed"
```

Comment:
Can we extract the release.yaml location to a variable and use it in `install_pipeline_crd` above too? Hopefully this will help avoid issues in future where we upgrade and forget to update one of these.

Reply:
Good catch

Reply:
done